core/src/mmu/gb_mmu.rs

```diff
@@ -136,25 +135,30 @@ impl<'a, D:AudioDevice, G:GfxDevice, J:JoypadProvider> GbMmu<'a, D, G, J>{
             0xA000..=0xFDFF=>self.external_memory_bus.write(address, value),
             0xFE00..=0xFE9F=>self.io_bus.ppu.oam[(address-0xFE00) as usize] = value,
             0xFEA0..=0xFEFF=>{},
-            0xFF00..=0xFF4F | 
-            0xFF51..=0xFF6F |
-            0xFF71..=0xFF7F=>self.io_bus.write(address - 0xFF00, value),
-            0xFF50 | 0xFF70=>{
-                self.external_memory_bus.write(address, value);
+            BOOT_REGISTER_ADDRESS => {
+                self.external_memory_bus.write_boot_reg(value);
+                if !self.external_memory_bus.in_boot(){
+                    self.io_bus.set_boot_finished();
+                }
+            },
+            SVBK_REGISTER_ADRRESS => if self.mode == Mode::CGB && self.io_bus.is_cgb_enabled() {
```

Comment:
Is the check for the cgb enable really necessary? if the register does not have effect in DMG comp mode I can just refactor it.

Even if it does have effect maybe I can verify it inside the fucntion call?

Reply:
The check for the mode was unnecessary, the problem here is that writes to this reg could affect DMG comp mode